heron/tools/common/src/python/utils/config.py

```diff
@@ -151,12 +165,12 @@ def get_heron_dir_explorer():
   From heron-cli with modification since we need to reuse cli's conf
   :return: root location for heron-cli.
   """
-  path_list = os.path.realpath(__file__).split('/')[:-10]
+  go_above_dirs = 10
```

Comment:
actually, i realized `heron-explorer` was moved to the same directory as `heron`. This means this method is no longer needed.